frame/beefy-mmr/src/lib.rs

```diff
@@ -124,6 +127,12 @@ pub mod pallet {
 		type BeefyDataProvider: BeefyDataProvider<Self::LeafExtra>;
 	}
 
+	/// Details of current BEEFY authority set.
```

Comment:
Do we need to fill this on Rococo (do a storage upgrade), or it isn't that important? The companion is currently missing any storage migrations

Reply:
It isn't that important, it'll be used for efficiency reasons by light clients, with no bad side-effects when not available.
It will self-populate on the first session change, in the meantime dummy default is fine.